src/test/java/org/mule/extension/api/runtime/FixedContentMetadataTestCase.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.api.runtime;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+
+public class FixedContentMetadataTestCase extends AbstractContentMetadataTestCase
+{
+
+    @Override
+    protected ContentMetadata createContentMetadata()
+    {
+        return new FixedContextMetadata(inputContentType, outputContentType);
+    }
+
+    @Test(expected = UnsupportedOperationException.class)
+    public void setOutputContentType()
+    {
+        contentMetadata.setOutputContentType(inputContentType);
+    }
+
+    @Test
+    public void isModifiable()
```

Comment:
Move this test to the abstract class and obtain the expected value using a method/field